src/requirements.txt

```diff
@@ -5,6 +5,6 @@ pyfiglet==0.8.post1
 reprint==0.5.2
 ruamel.yaml==0.16.7
 boto3==1.11.11
-black==23.3.0
+black
```

Comment:
Why do we need to unpin this?

Reply:
I could revert this. I thought that since we have added the style check to the PR, local black changes might differ from the CICD test if pinned for too long.